src/cbmc/bmc.cpp

```diff
@@ -173,16 +173,15 @@ void bmct::report_success()
     {
       xmlt xml("cprover-status");
       xml.data="SUCCESS";
-      std::cout << xml;
-      std::cout << "\n";
+      result() << preformatted_output << xml << eom;
     }
```

Comment:
Same here -- no XML into the message streams. In the extreme case, the message stream might then produce, say, JSON, which will then have escaped XML in it.